server/emails/templates/CommentMentionedEmail.tsx

```diff
@@ -92,8 +92,8 @@ export default class CommentMentionedEmail extends BaseEmail<
     );
   }
 
-  protected subject({ actorName, document }: Props) {
-    return `${actorName} mentioned you in “${document.title}”`;
+  protected subject({ document }: Props) {
+    return `Mentioned you in “${document.title}”`;
```

Comment:
This matches the `DocumentMentionedEmail` subject now - helps with threading in Gmail.
Lmk if this needs to be reverted.